cypress.config.ts

```diff
@@ -1,7 +1,8 @@
-import { getRequiredEnv, toInteger } from '@fullstacksjs/toolbox';
+import { getEnv, toInteger } from '@fullstacksjs/toolbox';
 import { defineConfig } from 'cypress';
 
-const port = toInteger(getRequiredEnv('PORT'), 3000);
+const port = toInteger(getEnv('PORT', ''), 3000);
```

Comment:
But It was required before :thinking: no it's optional

Reply:
another issue is that we're not loading the port on `cypress run` I'll add the env load to it too